### PR TITLE
fix: Fixed presence state for Tuya RT_ZCZ03Z (_TZE204_uxllnywp)

### PR DESCRIPTION
**Pull request by Zuz666**
Fixed presence state for Tuya RT_ZCZ03Z (_TZE204_uxllnywp).
Works fine now. ;)

### CODE REVIEW

**Comment by Zuz666** [comment]
I'm not sure why Lint failed to run on the file called 'ikea.ts', which I didn't modify. I'm not sure if it's because of a recent pull request (PR) or if I need to take some action in my PR to fix the issue.

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -11953,7 +11953,7 @@ const definitions: DefinitionWithExtend[] = [
         ],
         meta: {
             tuyaDatapoints: [
-                [1, 'presence', tuya.valueConverter.trueFalse1],
+                [1, 'presence', tuya.valueConverterBasic.lookup({ True: 4, False: 0 })],
```
Comment (Koenkk) [inline src/devices/tuya.ts:11956]:
```suggestion
                [1, 'presence', tuya.valueConverterBasic.trueFalse(4)],
```

Can you check if this works?

Comment (Zuz666) [reply src/devices/tuya.ts:11956]: Works like a charm , though.
Thanks!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**